lib/rest_5.js

```diff
@@ -466,9 +466,9 @@ function* resolveResult(result) {
     var count = 0;
```

Comment:
I think we should just not be duplicating ourselves at all:
```
function *resolveResult(result) {
  return (yield resolveResults([result]))[0];
}
```
We can add an assertion that the length of the resolveResults return value is 1, but that's optional.